projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-quote.ts

```diff
@@ -1758,3 +1758,60 @@ export function reload() {
 function log(comment: string, functionName: string) {
   cy.log(`##### ${comment} <${functionName}> #####`);
 }
+
+/**
+ * Registers read vendor quote route.
+ */
+export function registerReadVendorQuoteRoute() {
+  log('Registers read vendor quote route.', registerReadVendorQuoteRoute.name);
+  cy.intercept({
+    method: 'GET',
+    path: `${Cypress.env('OCC_PREFIX')}/${SHOP_NAME}/users/*/quotes*`,
+  }).as(READ_VENDOR_QUOTE.substring(1)); // strip the '@'
+}
+
+/**
+ * Navigates to the vendor quote list via my account.
+ */
+export function navigateToVendorQuoteListFromMyAccount() {
+  log(
+    'Navigates to the vendor quote list via my account',
+    navigateToVendorQuoteListFromMyAccount.name
+  );
+  cy.get('.accNavComponent')
+    .should('contain.text', 'My Account')
+    .and('be.visible')
+    .within(() => {
+      cy.get('nav > ul > li > button').first().focus().trigger('keydown', {
+        key: ' ',
+        code: 'Space',
+        force: true,
+      });
+      cy.get('cx-generic-link')
+        .contains('Quotes')
+        .should('be.visible')
+        .click()
+        .then(() => {
+          cy.wait(READ_VENDOR_QUOTE);
+          cy.url().should('include', 'quotes');
+        });
+    });
+}
+/**
+ * Navigates to the quote with status as Vendor Quote.
+ */
+export function navigateToVendorQuote() {
```

Comment:
Please, remove ths method because this method already exists, see b2b-quote.ts line 296: `navigateToQuotesList`